README_WINDOWS.md

```diff
@@ -335,12 +303,11 @@ the following folder structure *exactly*:
             bin
             include
             lib
-        readline
-            bin
-            include
-            lib
         fftw
 
+(Note: `libsndfile`'s installed files are by default placed in
+`C:\Program Files\Mega-Nerd\libsndfile`.)
```

Comment:
this could be placed above to be less verbose - line 296 in the current revision: "Move the installed files of `libsndfile` (by default, installed to `C:\Program Files`..."